charts/supabase/values.example.yaml

```diff
@@ -1,180 +1,144 @@
-jwt:
-  secretName: "demo-supabase-jwt"
-
-smtp:
-  secretName: "demo-supabase-smtp"
+secret:
+  jwt:
+    anonKey: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyAgCiAgICAicm9sZSI6ICJhbm9uIiwKICAgICJpc3MiOiAic3VwYWJhc2UtZGVtbyIsCiAgICAiaWF0IjogMTY0MTc2OTIwMCwKICAgICJleHAiOiAxNzk5NTM1NjAwCn0.dc_X5iR_VP_qT0zsiyj_I_OZ2T9FtRU2BBNWN8Bu4GE
+    serviceKey: eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyAgCiAgICAicm9sZSI6ICJzZXJ2aWNlX3JvbGUiLAogICAgImlzcyI6ICJzdXBhYmFzZS1kZW1vIiwKICAgICJpYXQiOiAxNjQxNzY5MjAwLAogICAgImV4cCI6IDE3OTk1MzU2MDAKfQ.DaYlNEoUrrEn2Ig7tqibS-PHK5vgusbcbo7X36XVt4Q
+    secret: your-super-secret-jwt-token-with-at-least-32-characters-long
+  smtp:
+    username: your-mail@example.com
+    password: example123456
+  dashboard:
+    username: supabase
+    password: this_password_is_insecure_and_should_be_updated
+  db:
+    username: postgres
+    password: example123456
+    database: postgres
+  analytics:
+    apiKey: your-super-secret-and-long-logflare-key
```

Comment:
Would be very much appreciated if this can also handle references to secrets, such that the `values.yaml` of a user can also be stored in their code repo (without it containing passwords). Especially when using GitOps for Kubernetes deployments this would be important.

Reply:
I'm planning to add back reference to existing secrets, hopefully after this PR is merged. Otherwise, this PR might be delayed yet again.

This is a preview code for referencing existing secrets https://github.com/drpsyko101/supabase-kubernetes/commit/138e6f9991c09e702128042fc64fdbef259b6bed . Please note that this is not fully tested yet. Some special characters might not be encoded correctly, especially for postgres URL.

Reply:
Thanks for the update! And I highly appreciated your efforts!